ctms/crud.py

```diff
@@ -57,6 +57,15 @@
 from .schemas.base import BaseModel
 
 
+def count_total_contacts(db: Session):
+    query = text(
+        "SELECT reltuples AS estimate "
+        "FROM pg_class "
+        f"where relname = '{Email.__tablename__}'"
```

Comment:
Should we add a comment here describing why we're not doing `SELECT COUNT(*) FROM emails`?